R/FilteredDataset.R

```diff
@@ -341,9 +340,9 @@ FilteredDataset <- R6::R6Class( # nolint
     #' UI module to add filter variable for this dataset.
     #'
     #' @param id (`character(1)`)
-    #'  identifier of the element - preferably containing dataset name
+    #'   `shiny` module instance id.
     #'
-    #' @return function - `shiny` UI module
+    #' @return `shiny.tag`
```

Comment:
This does not return. You have a nice description elsewhere:
```suggestion
    #' @return Virtual method, returns nothing and raises error.
```